modules/backup-and-restore/pages/gbar-legacy.adoc

```diff
@@ -0,0 +1,273 @@
+= Legacy Backup and Restore
+:page-aliases: backup-and-restore.adoc
+
+Graph Backup And Restore (GBAR), is an integrated tool for backing up and restoring the data and data dictionary (schema, loading jobs, and queries) of a TigerGraph instance or cluster.
+
+The backup feature packs TigerGraph data and configuration information into a directory on the local disk or a remote AWS S3 bucket.
+Multiple backup files can be archived.
+Later, you can use the restore feature to roll back the system to any backup point.
+This tool can also be integrated easily with Linux cron to perform periodic backup jobs.
+
+[NOTE]
+====
+The current version of GBAR is intended for restoring the same machine that was backed up. For help with cloning a database (i.e., backing up machine A and restoring the database to machine B), please https://tigergraph.zendesk.com/hc/en-us/[open a support ticket].
+====
+
+== Synopsis
+
+
+[source,text]
+----
+Usage: gbar backup [options] -t <backup_tag>
+		      gbar restore [options] <backup_tag>
+		      gbar list [backup_tag] [-j]
+		      gbar remove|rm <backup_tag>
+		      gbar cleanup
+		      gbar expand [-a] <new_nodes>
+		           New nodes must be written in <name>:<host> pairs separated by comma
+		           Example:
+		               m1:192.168.1.2,m2:192.168.1.3,m3:192.168.1.4
+
+		Options:
+		 -h, --help     	Show this help message and exit
+		 -v             	Run with debug info dumped
+		 -vv            	Run with verbose debug info dumped
+		 -y             	Run without prompt
+		 -j            		Print gbar list as JSON
+		 -t BACKUP_TAG  	Tag for backup file, required on backup
+		 -a, --advanced    Enable advanced mode for node expansion
+----
+
+
+
+The `-y` option forces GBAR to skip interactive prompt questions by selecting the default answer.
+There is currently one interactive question:
+
+* At the start of restore, GBAR will always ask if it is okay to stop and reset the TigerGraph services: (y/N)? The default answer is yes.
+
+== Configure GBAR
+
+Before using the backup or the restore feature, GBAR must be configured.
+
+. Run `gadmin config entry system.backup`.
+At each prompt, enter the appropriate values for each config parameter.
++
+[source,console]
+----
+$ gadmin config entry system.backup
+
+System.Backup.TimeoutSec [ 18000 ]: The backup timeout in seconds
+New: 18000
+
+System.Backup.CompressProcessNumber [ 8 ]: The number of concurrent process for compression during backup
```

Comment:
I am not sure that this gbar-legacy should change, since the new one has changed.

> System.Backup.CompressProcessNumber [ 0 ]: The number of concurrent processes for compression during backup. Value 0 means the number of processes used to compress equals the node CPU's cores.